deploy/examples/kiali/kiali-operator.yaml

```diff
@@ -0,0 +1,398 @@
+  resources:
+  - adapters
+  - apikeys
+  - bypasses
+  - authorizations
+  - checknothings
+  - circonuses
+  - cloudwatches
+  - deniers
+  - dogstatsds
+  - edges
+  - fluentds
+  - handlers
+  - instances
+  - kubernetesenvs
+  - kuberneteses
+  - listcheckers
+  - listentries
+  - logentries
+  - memquotas
+  - metrics
+  - noops
+  - opas
+  - prometheuses
+  - quotas
+  - quotaspecbindings
+  - quotaspecs
+  - rbacs
+  - redisquotas
+  - reportnothings
+  - rules
+  - signalfxs
+  - solarwindses
+  - stackdrivers
+  - statsds
+  - stdios
+  - templates
+  - tracespans
+  - zipkins
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - watch
+- apiGroups: ["networking.istio.io"]
+  resources:
+  - destinationrules
+  - gateways
+  - serviceentries
+  - sidecars
+  - virtualservices
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - watch
+- apiGroups: ["authentication.istio.io"]
+  resources:
+  - meshpolicies
+  - policies
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - watch
+- apiGroups: ["rbac.istio.io"]
+  resources:
+  - clusterrbacconfigs
+  - rbacconfigs
+  - servicerolebindings
+  - serviceroles
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - watch
+- apiGroups: ["apps.openshift.io"]
+  resources:
+  - deploymentconfigs
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: ["project.openshift.io"]
+  resources:
+  - projects
+  verbs:
+  - get
+- apiGroups: ["route.openshift.io"]
+  resources:
+  - routes
+  verbs:
+  - get
+- apiGroups: ["monitoring.kiali.io"]
+  resources:
+  - monitoringdashboards
+  verbs:
+  - get
+  - list
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kiali-operator
+  labels:
+    app: kiali-operator
+    version: v0.21.0
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kiali-operator
+  labels:
+    app: kiali-operator
+    version: v0.21.0
+subjects:
+- kind: ServiceAccount
+  name: kiali-operator
+  namespace: kiali-operator
+roleRef:
+  kind: ClusterRole
+  name: kiali-operator
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kiali-operator
+  labels:
+    app: kiali-operator
+    version: v0.21.0
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kiali-operator
+      version: v0.21.0
+  template:
+    metadata:
+      name: kiali-operator
+      labels:
+        app: kiali-operator
+        version: v0.21.0
+    spec:
+      serviceAccountName: kiali-operator
+      containers:
+      - name: ansible
+        command:
+        - /usr/local/bin/ao-logs
+        - /tmp/ansible-operator/runner
+        - stdout
+        image: quay.io/kiali/kiali-operator:v0.21.0
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+        - mountPath: /tmp/ansible-operator/runner
+          name: runner
+          readOnly: true
+      - name: operator
+        image: quay.io/kiali/kiali-operator:v0.21.0
```

Comment:
Is this going to be created via CR?

Reply:
This is the operator install, which should be moved out of istio-operator.  istio-operator should document how to install the kiali operator.  Eventually, we should be using OLM for the installs.

Reply:
i will remove all of these deploy/example files - they were just here to help people test/review the PR. But I will remove if no one is using them.